src/files/domain/models/FilePreview.ts

```diff
@@ -1,4 +1,5 @@
 import FileTypeToFriendlyTypeMap from './FileTypeToFriendlyTypeMap'
+import { FilePermissions } from './File'
```

Comment:
I noticed that the `FilePermissions` model is actually duplicated. There is already a `FileUserPermissions` model, we could delete the current `FilePermissions` from the File.ts file and then rename `FileUserPermissions`  to `FilePermissions` and then use it everywhere

Reply:
I also removed canEditDataset from the FilePermissions interface and also the enum, because it didn't seem to be needed, let me know if that's ok.